examples/nep_potentials/PbTe/train/nep.in

```diff
@@ -1,8 +1,10 @@
+version      3
 type         2 Te Pb
 version      3
 cutoff       8 4
 n_max        4 4
 l_max        4 2
 neuron       30
 batch        25
-generation   10000
+generation   1
+lambda_f     0
```

Comment:
restore this file before finishing the PR. You can copy the example for your use, but do not commit it here.